lib/discordrb/data/user.rb

```diff
@@ -132,6 +149,10 @@ def initialize(data, bot)
 
       @status = :offline
       @client_status = process_client_status(data['client_status'])
+      @banner_id = data['banner']
+      @system_account = data.key?('system') ? data['system'] : false
```

Comment:
This key based guard seems somewhat pointless during the saturation step. I would maybe just do `data['system'] || false` or `!!data['system']`